### DISCUSSION TITLE
PCI DSS 4.0

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#453] [Q&A] [answered]
Hi,

I've created the referential for PCI DSS 4.0

I tried to be exaustive, maybe too as Monarc makes a error when importing it...

[PCI-DSS 4.0 Monarc.xlsx](https://github.com/monarc-project/MonarcAppFO/files/9281480/PCI-DSS.4.0.Monarc.xlsx)

Maybe the fields label is too large for Monarc?

is it possible?

Regards,

**Comment by cedricbonhomme** [comment, marked as answer]
Hi,

Thanks !

I do not think the fields are too large for MONARC. Maybe the display won't be ideal. I have been able to test only with one element, it looked correct.

Are you sure about the codes of in the xlsx file ? There are duplicates. Is this normal ? They must be unique for MONARC.

Also for example there is the code:
'1.1
With the apostrophe.

**Reply by danje57** [reply]
I've simplified the format without the audit suggestion.

Here it is!!!

[allControls_PCI DSS 4.0.csv](https://github.com/monarc-project/MonarcAppFO/files/9283797/allControls_PCI.DSS.4.0.csv)
!

**Reply by danje57** [reply]
Feel free to share as usual. Now I'll make a break on these ;)

**Reply by cedricbonhomme** [reply]
@danje57 thank you for all your contributions !

The object is here:
https://objects.monarc.lu/object/view/5238
It can correctly be imported in MONARC. Maybe an idea for the future is to cut the very long labels in MONARC in some views (only visually, and the full string could still be visible when hovering the label with the mouse).

you can mark the discussion as resolved if fine for you ;-)

**Reply by danje57** [reply]
Thanks to you too.

I'm ok with your suggestion too, very long label is not a good option and difficult to read.

I'll mark the post as solved! Thanks again!!!
